apps/backend/src/libs/modules/cloud-formation/libs/templates/ec2-instance-template.json

```diff
@@ -39,11 +41,27 @@
 					"Ref": "ImageId"
 				},
 				"InstanceType": "t3.micro",
+				"KeyName": "meetlytic-ssh",
 				"SecurityGroupIds": [
 					{
 						"Ref": "MeetlyticSecurityGroup"
 					}
 				],
+				"UserData": {
+					"Fn::Base64": {
+						"Fn::Sub": [
+							"#!/bin/bash\n\n# --- 1. Install System Dependencies ---\nsudo apt update -y\nsudo apt install -y git\ncurl -fsSL https://deb.nodesource.com/setup_22.x | sudo -E bash -\nsudo apt install -y nodejs\n\n# Install Chromium shared libraries for Puppeteer\nsudo apt-get install -y libatk-bridge2.0-0 libgtk-3-0 libappindicator3-1 libnss3 libxss1 libasound2 libgbm1 libx11-xcb1 libxcomposite1 libxkbcommon0\n\n# --- 2. Clone the Repository and Switch Branch ---\ncd /home/ubuntu\ngit clone --branch 43-feat-join-zoom-meetings https://github.com/BinaryStudioAcademy/bsa-2025-meetlytic.git\n\n# --- 3. Install NPM Dependencies & Puppeteer Browser ---\ncd bsa-2025-meetlytic\n/usr/bin/npm install\n/usr/bin/npx puppeteer browsers install chrome\n\n# --- 4. Configure and Run the Bot ---\ncd apps/bot\n\necho \"NODE_ENV=production\" > .env\necho \"MEETING_ID=${MeetingId}\" >> .env\necho \"MEETING_PASSWORD=${MeetingPassword}\" >> .env\necho \"BOT_NAME=Meetlytic\" >> .env\necho \"OPEN_AI_KEY=\" >> .env\necho \"TRANSCRIPTION_MODEL=whisper-1\" >> .env\n\nnohup /usr/bin/npm run start:dev > /home/ubuntu/bot.log 2>&1 &",
```

Comment:
As I see all .env properties of the bot should be added here. After this is merged, please, inform everyone about this